postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -360,19 +371,31 @@ def _run_explain(self, dbname, statement, obfuscated_statement):
                 return None
             return result[0][0]
 
-    def _run_explain_safe(self, dbname, statement, obfuscated_statement):
-        # type: (str, str, str) -> Tuple[Optional[Dict], Optional[DBExplainError], Optional[Exception]]
+    def _run_explain_safe(self, max_query_size, dbname, statement, obfuscated_statement):
+        # type: (int, str, str, str) -> Tuple[Optional[Dict], Optional[DBExplainError], Optional[str]]
         if not self._can_explain_statement(obfuscated_statement):
             return None, DBExplainError.no_plans_possible, None
 
+        if self._is_statement_truncated(max_query_size, statement):
+            self._check.count(
+                "dd.postgres.statement_samples.error",
+                1,
+                tags=self._dbtags(dbname, "error:explain-{}".format(DBExplainError.statement_truncated)),
+            )
+            return (
+                None,
+                DBExplainError.statement_truncated,
+                "statement truncated with track_activity_query_size={}".format(max_query_size),
```

Comment:
Nit: `statement truncated with` is redundant because the error code is already `statement_truncated`

Reply:
✅